Source/inv.cpp

```diff
@@ -1174,7 +1157,8 @@ bool CanBePlacedOnBelt(const Item &item)
 {
 	return FitsInBeltSlot(item)
 	    && item._itype != ItemType::Gold
-	    && item._iStatFlag
+	    && MyPlayer->GetItemLocation(item) == ILOC_UNEQUIPABLE
+	    && MyPlayer->CanUseItem(item)

```

Comment:
I'm not sure this is needed, copied it from `CheckInvPaste()` for now.

Reply:
`CanUseItem()` is for scrolls and runes. The `ILOC_UNEQUIPABLE` thing appears to be superfluous, unless there's a usable ring or amulet in the game that I don't know about. 🤷

Reply:
`CanUseItem()` should also be unessesery if `_iStatFlag` is up to date, but we are also moving away from it, so might as well use `CanUseItem()` for safety, even if the current way I did it is a bit ugly :)